device-registry/src/main/resources/db/migration/V5__system_info.sql

```diff
@@ -0,0 +1,7 @@
+
+CREATE TABLE DeviceSystem (
+  uuid char(36) NOT NULL,
+  system_info LONGTEXT NOT NULL,
```

Comment:
This should probably be a `json` in mysql and a dynamic col in maria db. As we currently use mariadb for dev, but mysql for prod, not sure what we can do here, but probably we should unifity the dbs we use and change this column before adding much more properties here.